cmd/eib/main.go

```diff
@@ -92,12 +94,12 @@ func validateImageConfigDir(configDir string) error {
 func main() {
 	imageConfig, buildConfig, err := processArgs()
 	if err != nil {
-		log.Error(err)
+		zap.L().Error("parsing CLI arguments", zap.Error(err))
```

Comment:
The message is suitable for error wrapping, however when it comes to logging we usually want to provide a complete message / event e.g. `processing CLI arguments failed`.